pkg/nanopb/nanopb.inc.mk

```diff
@@ -15,6 +15,7 @@ $(GENSRC): $(PROTOBUF_FILES)
 	$(Q)D=$(BINDIR)/$(MODULE) && \
 	  mkdir -p "$$D" && \
 		cd $(CURDIR) && \
+		make -C $(PKGDIRBASE)/nanopb/generator/proto && \
```

Comment:
can you guard this with something like ```if $PROTOC_GEN_NANOPB == $(PKGDIRBASE)/nanopb/generator/protoc-gen-nanopb```, so that make will only be called if the user doesn't supply an external version?

Or is that overkill...?

Reply:
The question is why would the user ever want to provide an external version?

I can add such check, but I don't quite see the use case for it yet.

Reply:
ok, fine with me!